src/components/Player/Header/PlayerHeader.jsx

```diff
@@ -107,78 +150,64 @@ const getRegistrationBadge = registered => registered && (
   />
 );
 
-const MedalStyled = styled.div`
-  .rankMedal {
-    position: relative;
-    height: 37px;
-    width:37px;
-    top: 3px;
-    margin-right: 5px;
-    text-align: center;
-    &[data-hint-position="top"] {
-      &::after {
-        margin-bottom: 3px;
-        margin-left: -37px;
-      }
-  
-      &::before {
-        top: -3px;
-        margin-left: 13px;
-      }
-    }
-    &-icon {
-      top: 0px;
-      height: 100%;
-      width: 100%;
-    }
-    &-star {
-      position: absolute;
-      left: 0px;
-      height: 100%;
-      width: 100%;
-    }
-  }
-`;
-
-const getRankTierMedal = (rankTier) => {
+const getRankTierMedal = (rankTier, leaderboardRank) => {
   let medalElement = null;
   const imgDescription = rankTierToString(rankTier);
   if (rankTier && rankTier > 9) {
-    const intRankTier = parseInt(rankTier, 10);
-    const iconPath = `/assets/images/dota2/rank_icons/rank_icon_${parseInt(intRankTier / 10, 10)}.png`;
-    const star = parseInt(intRankTier % 10, 10);
-    let correctStar = 0;
-    if (star <= 0) {
-      correctStar = 0;
-    } else if (star >= 5) {
-      correctStar = 5;
+    if (rankTier === 76) {
+      let iconPath = '';
+      if (leaderboardRank > 100) {
+        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7a.png';
+      } else if (leaderboardRank > 10) {
+        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7b.png';
+      } else {
+        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7c.png';
```

Comment:
won't this cause 7c to display if there is no leaderboard rank?  Should 7a be the default?

Reply:
That's my fault, 7a should be the default.
But I think if rank tier is 76, leaderboard rank will be exist certainly. So I wrote `if(rankTier === 76)` in line 157

Reply:
I change some code 
```
if (rankTier === 76 && leaderboardRank) {
      let iconPath = '/assets/images/dota2/rank_icons/rank_icon_7a.png'; // Divine Elite
      if (leaderboardRank <= 10) {
        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7c.png'; // Divine Top 10
      } else if (leaderboardRank <= 100) {
        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7b.png'; // Divine Top 100
      }
      ...
}
```